cashctrl_api/client.py

```diff
@@ -375,12 +343,48 @@ def list_files(self) -> pd.DataFrame:
         """
         files = pd.DataFrame(self.get("file/list.json")['data'])
         columns_except_path = {key: value for key, value
-                               in self.FILE_COLUMNS.items() if key != 'path'}
+                               in FILE_COLUMNS.items() if key != 'path'}
         df = enforce_dtypes(files, columns_except_path)
         if len(df) > 0:
             categories = self.list_categories('file')[['path', 'id']]
             categories = categories.rename(columns={'id': 'categoryId'})
             df = df.merge(categories, on='categoryId', how='left')
             df['path'] = df['path'].fillna('') + '/' + df['name']
-        df = enforce_dtypes(df, self.FILE_COLUMNS)
+        df = enforce_dtypes(df, FILE_COLUMNS)
         return df.sort_values('path')
+    
+    def list_tax_rates(self) -> pd.DataFrame:
+        """
+        List remote tax rates with their attributes.
+
+        Returns:
+            pd.DataFrame: A DataFrame with CashCtrlClient.TAX_COLUMNS schema.
+        """
+        tax_rates = pd.DataFrame(self.get("tax/list.json")['data'])
+        df = enforce_dtypes(tax_rates, TAX_COLUMNS)
+
+        return df.sort_values('value')
+    
+    def list_accounts(self) -> pd.DataFrame:
+        """
+        List remote accounts with their attributes.
+
+        Returns:
+            pd.DataFrame: A DataFrame with CashCtrlClient.ACCOUNT_COLUMNS schema.
+        """
+        accounts = pd.DataFrame(self.get("account/list.json")['data'])
+        df = enforce_dtypes(accounts, ACCOUNT_COLUMNS)
+
+        return df.sort_values('number')
+    
+    def list_journal_entries(self) -> pd.DataFrame:
+        """
+        List remote journal entries with their attributes.
+
+        Returns:
+            pd.DataFrame: A DataFrame with CashCtrlClient.ACCOUNT_COLUMNS schema.
+        """
+        journal_entries = pd.DataFrame(self.get("journal/list.json")['data'])
+        df = enforce_dtypes(journal_entries, JOURNAL_ENTRIES)
+
+        return df.sort_values('createdBy')
```

Comment:
Please sort by the field shown as 'date' in the GUI (don't know what it translates to).

Reply:
fixed

Reply:
The field 'date' in the GUI appears in `cc_client.list_journal_entries()` as column 'dateAdded'. 
You can verify by entering journal entries with various dates in the gui an then inspecting the return value of `cc_client.list_journal_entries()`.
(Not exactly a good naming scheme, but nothing we can change).

We should sort by 'dateAdded', instead of 'createdBy'.

Reply:
I agree, you are right